spinner/build.sh

```diff
@@ -48,16 +48,31 @@ function build_service_liferay {
 
 	cp ../../orca/templates/liferay/resources/usr/local/liferay/scripts/pre-startup/10_wait_for_dependencies.sh build/liferay/resources/usr/local/liferay/scripts/pre-startup
 
-	local hotfix=$(retrieve_descriptor_data "hotfix")
+	local branch=$(git -C "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}" rev-parse --abbrev-ref HEAD)
```

Comment:
@pedropaccola, can you try to use the Git parameters full name instead of the abbreviated versions of it?

For example, is there any full name for the parameter `-C`? Is it `--git-dir`?

BChan usually prefers to use the parameters in a more verbose way to make the code more meaningful since he may not understand all its logic :slightly_smiling_face: 

In addition, can you please try to sort the parameters according the alphabetic order, if it won't affect the command execution?